### PR TITLE
Documentation update.

### PR DESCRIPTION
**Pull request by morrowc**
I believe you say: "Step" but mean "Next" since there is no Step()
function.

### CODE REVIEW

**Review by corny — APPROVED** [review]

**Comment by morrowc** [comment]
"2 workflows awaiting approval
First-time contributors need a maintainer to approve running workflows"

I don't know whom counts for 'maintainer' in this case... or if the maintainer (corny?) must click something to make the workflows run?

**Review by jlaffaye — APPROVED** [review]

**Pull request merged**
